static/app/components/events/autofix/autofixSolution.tsx

```diff
@@ -658,12 +659,10 @@ export function AutofixSolution(props: AutofixSolutionProps) {
     );
   }
 
-  const changesDisabled = props.repos.every(repo => repo.is_readable === false);
-
   return (
     <AnimatePresence initial={props.isSolutionFirstAppearance}>
       <AnimationWrapper key="card" {...cardAnimationProps}>
-        <AutofixSolutionDisplay {...props} changesDisabled={changesDisabled} />
+        <AutofixSolutionDisplay {...props} />
```

Comment:
what are you doing with `changesDisabled` if not passing it here? is it an unused var?

Reply:
the above code for `changesDisabled` is inside `AutofixSolutionDisplay`